docs/conch/examples/sshsimpleserver.py

```diff
@@ -224,15 +229,6 @@ def getPrimes(self):
         return PRIMES
 
 
-portal = portal.Portal(ExampleRealm())
-passwdDB = InMemoryUsernamePasswordDatabaseDontUse()
-passwdDB.addUser(b'user', b'password')
-sshDB = SSHPublicKeyChecker(InMemorySSHKeyDB(
-    {b'user': [keys.Key.fromFile(CLIENT_RSA_PUBLIC)]}))
-portal.registerChecker(passwdDB)
-portal.registerChecker(sshDB)
-ExampleFactory.portal = portal
```

Comment:
when reading the code I was very confused about how ExampleFactory knew about the ExampleRealm and the various checkers.

IMHOP it's very unusual to define class properties outside of the classdef, so I missed this line completely